package.json

```diff
@@ -53,6 +53,7 @@
     "eslint": "^6.8.0",
     "eslint-config-react-app": "^5.0.1",
     "eslint-config-react-tools": "^1.1.7",
+    "eslint-plugin-flowtype": "^4.7.0",
```

Comment:
Do we need this? We aren't using flow in this project at the moment.

Reply:
Yep, for some reason without this, the linter will scream on VSCode; I confirmed this with Andy earlier today.